JSQMessagesDemo/DemoModelData.m

```diff
@@ -135,9 +135,12 @@ - (void)loadFakeMessages
      *  Setting to load extra messages for testing/demo
      */
     if ([NSUserDefaults extraMessagesSetting]) {
-        NSArray *copyOfMessages = [self.messages copy];
-        for (NSUInteger i = 0; i < 4; i++) {
-            [self.messages addObjectsFromArray:copyOfMessages];
+        NSArray *copyOfMessages = [NSKeyedUnarchiver unarchiveObjectWithData:
+            [NSKeyedArchiver archivedDataWithRootObject:self.messages]];
+        [self.messages addObjectsFromArray:copyOfMessages];
+        for (NSUInteger i = 0; i < 3; i++) {
+            [self.messages addObjectsFromArray:[NSKeyedUnarchiver unarchiveObjectWithData:
+                [NSKeyedArchiver archivedDataWithRootObject:copyOfMessages]]];
```

Comment:
hey @algrid, is there any issues if you combine inside the `for` loop and use `4` as the terminated number? That's more clear.